config/content.go

```diff
@@ -0,0 +1,6 @@
+package config
+
+type ContentConfig struct {
+	Disable       bool `json:",omitempty"`
+	GlobalDisable bool `json:",omitempty"` // not valid for shuttle
```

Comment:
please let us use the opportunity to name these configs so they are intuitive. In this case, using the current names `localContentAddingDisabled` and `contentAddingDisabled` is more discriptive

Reply:
No problem.